src/Microsoft.Azure.CosmosRepository.AspNetCore/Microsoft.Azure.CosmosRepository.AspNetCore.csproj

```diff
@@ -37,31 +36,30 @@
         <RepositoryUrl>https://github.com/IEvangelist/azure-cosmos-dotnet-repository</RepositoryUrl>
         <PackageLicenseFile>LICENSE</PackageLicenseFile>
         <RepositoryType>git</RepositoryType>
+        <Nullable>enable</Nullable>
     </PropertyGroup>
-
     <ItemGroup>
-        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
         <PackageReference Include="MinVer">
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="Scrutor" />
+        <PackageReference Include="AspNetCore.HealthChecks.CosmosDb"  />
+    </ItemGroup>
+    <ItemGroup>
```

Comment:
@IEvangelist spoke with @philip-reed around this change, and I am not sure when I implemented this the first time why I didn't just do a framework reference as this was the idea of the .AspNetCore package. I think this is good and not a problem, however, I just wonder if it's considered a breaking change or not?

Reply:
I don't believe it's a breaking change, it just adds a new dependency - but I could be wrong. I'm good with this...